### PR TITLE
Splash screen for Playground

### PR DESCRIPTION
**Pull request by bbenetskyy**
Added new Splash Screen.

This is not a typical splash screen - we hold initializing everything except Xamarin.Forms and initialize inside first "SplashScreenPage" while for user shown animated hexagon with Maric Gradients Label. All that controls have animated background gradients done with power of Magic Gradients:

https://user-images.githubusercontent.com/8330262/136698650-3e8f7da9-11d9-41fc-b98e-a835674fc41c.mov

### CODE REVIEW

File: Playground/Playground/App.xaml.cs [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
 namespace Playground
 {
     public partial class App : Application
     {
         public App()
         {
-            Device.SetFlags(new[] { "Shell_UWP_Experimental" });
-
             InitializeComponent();
+#if WINDOWS_UWP
```
[the remark this answers is not in the document]
Comment (bbenetskyy) [reply Playground/Playground/App.xaml.cs:10]: not it's corrected

File: Playground/Playground/Features/Home/HomePage.xaml [hunk 2]
```diff
@@ -8,6 +8,7 @@
              xmlns:ic="clr-namespace:Playground.Resources.Fonts;assembly=Playground"
              x:Class="Playground.Features.Home.HomePage"
              Title="Home"
+             BackgroundColor="DodgerBlue"
```
Comment (mgierlasinski) [inline Playground/Playground/Features/Home/HomePage.xaml:11]: What is DodgerBlue? :)

File: Playground/Playground/Playground.csproj [hunk 3]
```diff
@@ -16,6 +16,7 @@
 
   <ItemGroup>
     <EmbeddedResource Include="Resources\Fonts\IcoMoon.ttf" />
+    <None Remove="Styles\Saira-Bold.ttf" />
```
Comment (mgierlasinski) [inline Playground/Playground/Playground.csproj:19]: Do we use this font anywhere?

**Review by mgierlasinski — APPROVED** [review]

File: Playground/Playground/Playground.csproj (hunk 3 above)
Comment (bbenetskyy) [reply Playground/Playground/Playground.csproj:19]: I remove that font in my last commit, maybe VS did not clean all references in csproj, going to remove it manually.

File: Playground/Playground/Features/Home/HomePage.xaml (hunk 2 above)
Comment (bbenetskyy) [reply Playground/Playground/Features/Home/HomePage.xaml:11]: default blue navigation bar color, for iOS at home page. this can be safely deleted

**Pull request merged**